config/db.config.js

```diff
@@ -0,0 +1,10 @@
+const mongoose = require('mongoose');
+const DB_URL = process.env.MONGOLAB_URI || "mongodb://127.0.0.1:27017/urlshortener"
```

Comment:
You had left your personal mongolab credentials here. Removed them for you to automatically create a local mongo collection if it doesn't exist.